core/jazz_environment-event-handler/index.js

```diff
@@ -30,7 +30,7 @@ var errorHandler = errorHandlerModule(logger);
 
 var processedEvents = [];
 var failedEvents = [];
-
+var safeDetails;
```

Comment:
Please declare in local scope since there is a chance for overlap

Reply:
safeDetails are used for many events ( initial-commit/create-branch and update-environment). safeDetails are created in events(initial-commit/create-branch) which is required in update-env event to store in DB. Hence created as global to file.

Reply:
Since these are a queue based system, if you declare this as global scope,  there is a chance for  overlapping with the other events. So if you need it in other places, declare it as local scope and do pass it to other methods

Reply:
Discussed and addressed.